### PR TITLE
Hotfix/bulk import improvements

### PR DESCRIPTION
**Pull request by gannetson**

### CODE REVIEW

**Comment by eodolphi** [comment]
In general:

How will this work after the import? How will new employees receive the custom fields? Or will we do an import on a regular basis?

File: bluebottle/utils/admin.py [hunk 1]
```diff
@@ -91,10 +94,34 @@ def export_as_csv(modeladmin, request, queryset):
...
+                for field in CustomMemberFieldSettings.objects.all():
+                    labels.append(field.name)
+            writer.writerow(row)
 
         for obj in queryset:
-            writer.writerow([prep_field(request, obj, field, manyToManySep) for field in field_names])
+            row = [prep_field(request, obj, field, manyToManySep) for field in field_names]
+            # Write extra field data
+            if queryset.model is Project:
+                for field in CustomProjectFieldSettings.objects.all():
```
Comment (eodolphi) [inline bluebottle/utils/admin.py]: Tying a general export function to project specific models seems like a bad idea. Any chance we can clean this up a bit?

**Comment by gannetson** [comment]
Employees can't update these fields themselves, for now. The platform manager can change this in admin. I don't think we want to do periodic updates/imports at this moment. If we want to have something like this we should do so through webhooks / API or such.

File: bluebottle/utils/admin.py (hunk 1 above)
Comment (gannetson) [reply bluebottle/utils/admin.py]: True

**Pull request merged**